README.md

```diff
@@ -418,6 +418,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [Node JS Architecture](https://www.journaldev.com/7462/node-js-architecture-single-threaded-event-loop)
 - [Deploy Node App on AWS EC2](https://jobin.hashnode.dev/deploy-your-node-js-app-into-production-with-nginx-and-ssl-installationstep-by-step-tutorial)
 - [30SecondsOfCode Node JS Snippets](https://www.30secondsofcode.org/js/t/node/p/1)
+- [Complete NodeJS introduction](https://codeburst.io/the-only-nodejs-introduction-youll-ever-need-d969a47ef219)
```

Comment:
Replace with the actual title of the post.

Reply:
I made a new PR #217 
Please review